src/NuProj.Packages/NuProj.nuproj

```diff
@@ -51,7 +51,10 @@ There is also a Visual Studio extension which you find under http://bit.ly/NuPro
     <Content Include="$(BasePath)raw\Additional\Microsoft.Common.NuProj.targets">
       <Link>tools\Microsoft.Common.NuProj.targets</Link>
     </Content>
-    <Content Include="..\packages\NuGet.CommandLine.2.8.6\tools\NuGet.exe">
+    <Content Include="..\NuGet.VSTS\tools\CredentialProvider.VSS.exe">
```

Comment:
I could not find such a package; as far as I could tell, it is only available on a VSTS feed. 
We could proceed one of three ways:

1. I could manually publish a NuGet package with the credential provider
2. Revert the change and let users manually overwrite the files themselves, or
3. Keep the binaries as-is

Let me know which you think would be best